content/en/tracing/troubleshooting/iis_dotnet_framework.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Troubleshooting - Not seeing .NET Framework traces
+kind: documentation
+---
+
+## Problem 
+
+.NET Framework APM traces are not showing within Datadog.
+
+## Solution
+
+If .NET Framework APM traces are not showing within Datadog, please review the possible reasons below.
+
+### Compatability 
+
+The .NET Tracer supports automatic instrumentation on .NET Framework 4.5 and above. 
+
+
+### Confirm Installation 
+
+#### Datadog Agent
+
+Has the Datadog Agent for Windows OS been installed locally? If not, please visit our [Datadog Windows agent][1] guide.
```

Comment:
```suggestion
Verify the Datadog Agent for Windows OS been installed locally. If not, visit the [Datadog Windows Agent][1] guide.
```

Reply:
General comment - @ruthnaebeck @jaycdave88 might it be better to remove the h4's and make this a list of two things under `Confirm installation`?  I feel like there's a lot of headers for not too much text for these steps.